site/public/js/server.js

```diff
@@ -1357,11 +1356,6 @@ $(document).ready(function() {
             }
         }
     });
-
-    //If they make their screen too small, collapse the sidebar to allow more horizontal space
-    $(document.body).resize(function() {
-        checkSidebarCollapse();
-    });
```

Comment:
To confirm, you're removing this because we have `window.addEventListener("resize", checkSidebarCollapse);` at the top of this file and that the two of them were accomplishing the same thing, and we only need the one. 

I think we might want to move that event listener attachment into the `$(document).ready(function() {` as I think calling `checkSidebarCollapse` before the DOM has loaded (or at least the sidebar of the DOM) will cause an error to be thrown when calling `$("aside").toggleClass` as `$("aside")` will return `null` and then calling `toggleClass` on `null`.

Reply:
Good comments.  I thought they were accomplishing the same thing but when I removed the event listener at the top of the file, everything broke, while removing the above event listener seemed to have no effect.  I'll move the event listener from the top of the file into the `document.ready()`.